### DISCUSSION TITLE
What is /secret-path ?, What am I supposed to put there?

### DISCUSSION THREAD

**Discussion by coxato** [telegraf/telegraf#1356] [Q&A] [answered]
I see that a lot in webhooks examples, something like `bot.telegram.setWebhook('https://MY-APP.heroku.com/secret-path')` and `expressApp.use(bot.webhookCallback('/secret-path'))` so What am I supposed to put there?, maybe this: `/bot<<my_token>>` ???

**Comment by wojpawlik** [comment, marked as answer]
From https://core.telegram.org/bots/api#setwebhook:
> If you'd like to make sure that the Webhook request comes from Telegram, we recommend using a secret path in the URL, e.g. `https://www.example.com/<token>`. Since nobody else knows your bot's token, you can be pretty sure it's us.

When using [`Telegraf::launch`](https://telegraf.js.org/classes/telegraf.html#launch), `hookPath` defaults to random string: https://github.com/telegraf/telegraf/blob/a24ec036747e2cf462c8496a25b3a8755264b212/src/telegraf.ts#L170-L172

**Reply by coxato** [reply]
oh!, ok thanks :)
